lib/components/user/verify-email-screen.js

```diff
@@ -10,25 +16,57 @@ const _handleClick = () => window.location.reload()
  * (One way to make sure the parent page fetches the latest email verification status
  * is to simply reload the page.)
  */
-const VerifyEmailScreen = () => (
-  <div>
-    <h1>Verify your email address</h1>
-    <p>
-      Please check your email inbox and follow the link in the message
-      to verify your email address before finishing your account setup.
-    </p>
-    <p>
-      Once you're verified, click the button below to continue.
-    </p>
-
-    <Button
-      bsSize='large'
-      bsStyle='primary'
-      onClick={_handleClick}
-    >
-      My email is verified!
-    </Button>
-  </div>
-)
-
-export default VerifyEmailScreen
+class VerifyEmailScreen extends Component {
+  _handleResendVerificationEmail = () => {
+    const { auth0, resendVerificationEmail } = this.props
+    resendVerificationEmail(auth0)
+  }
+
+  _handleClick = () => window.location.reload()
+
+  render () {
+    return (
+      <div>
+        <h1>Verify your email address</h1>
+        <DivSpacer>
+          Please check your email inbox and follow the link in the message
+          to verify your email address before finishing your account setup.
+        </DivSpacer>
+        <p>
+          Once you're verified, click the button below to continue.
+        </p>
+        <DivSpacer>
+          <Button
+            bsSize='large'
+            bsStyle='primary'
+            onClick={this._handleClick}
```

Comment:
Since there are now 2 buttons, can this handler be renamed to something more specific?

Reply:
Good catch. Updated in https://github.com/opentripplanner/otp-react-redux/pull/227/commits/7152d277edd003953ccc3fb8d5921ebfd9f93b67.